addons/units/subStrings/bwc_msv.hpp

```diff
@@ -53,5 +53,5 @@ QBWC_MSV(demo),
 QBWC_MSV(vicc),
 QBWC_MSV(vicd),
 QBWC_MSV(pilot),
-QBWC_MSV(cc),
+QBWC_MSV(cc)
```

Comment:
It's wrong but for some reasons the game wants it
```
ErrorMessage: File z\potato\addons\units\subStrings\bwc_msv.hpp, line 57: Config: '#' encountered instead of ','
Application terminated intentionally


```

Reply:
hemtt doesn’t build with it, arma doesn’t launch without… love it